### ISSUE TITLE
Make GUI installer accessible for visually impaired users

### ISSUE THREAD

**Issue by leonardder** [osmc/osmc#249]
The QT GUI downloader/installer is currently not accessible using screen reading software for blind and visually impaired users. As a Blind OSMC user, I always have to manually extract disk images to SD-cards. It would be a great addition if efford could be made to implement accessibility. More about QT accessibility can be found in [this QT documentation](http://doc.qt.io/qt-4.8/accessible.html).

**Comment by samnazarko** [comment]
This is a good suggestion -- I will rewrite the installer soon and will see if we can leverage these features.

**Comment by leonardder** [comment]
@samnazarko: Are there still plans for an installation rewrite?
It seems it is quite easy to enable accessibility for the installer, consult [this](http://stackoverflow.com/questions/3893560/qt-accessible-widgets) question for more details.

**Comment by samnazarko** [comment]
Yes. It hasn't been done yet however because I do not fully know the requirements for the PC installer in my head yet (when we add x86_64 targets)
